lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -61,18 +61,22 @@ static llvm::cl::opt<bool> SILViewSILGenCFG(
     "sil-view-silgen-cfg", llvm::cl::init(false),
     llvm::cl::desc("Enable the sil cfg viewer pass before diagnostics"));
 
-llvm::cl::opt<bool> EnableDeinitDevirtualizer(
-    "enable-deinit-devirtualizer", llvm::cl::init(false),
-    llvm::cl::desc("Enable the DeinitDevirtualizer pass."));
+static llvm::cl::opt<bool>
+    EnableDestroyHoisting("enable-destroy-hoisting", llvm::cl::init(false),
```

Comment:
This looks like a wrong rebase: the EnableDestroyHoisting option was recently removed by Nate